content/blog/2019-08-15-new-react-devtools.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Introducing the New React DevTools"
+author: [bvaughn]
+---
+We are excited to announce a new release of the React Developer Tools, available today in Chrome, Firefox, and (Chromium) Edge!
+
+## What's changed?
+
+A lot has changed in version 4!
+At a high level, this new version should offer significant performance gains and an improved navigation experience.
```

Comment:
I still feel like there should be at least one flagship feature here. Maybe add "This release also offers full support for React Hooks, including inspecting nested objects."

Reply:
Sounds good